tools/IncludeBin/IncludeBin.cpp

```diff
@@ -0,0 +1,33 @@
+#include <cstdio>
+#include <cstdlib>
+
+int main(int argc, char **argv) {
+  if (argc != 3) {
+    fprintf(stderr, "Usage: bin-include input_file output_file\n");
+    exit(1);
+  }
```

Comment:
If we wanted to simplify the handling of the argument, we could make this read from stdin and write to stdout.